plugins/gh.ml

```diff
@@ -63,20 +64,17 @@ let reply_of_key key =
 
 let name = "Pull request linker"
 
-let rule m = m.Irc_message.command = "PRIVMSG" &&
-  match m.Irc_message.trail with
-  | Some msg -> Re.execp pullreq_re msg
-  | None -> false
+let rule = function
+  | {Irc_message.command = Irc_message.PRIVMSG (_, msg)} ->
+    Re.execp pullreq_re msg
+  | _ -> false
 
 let run ~connection ~message =
-  match message.Irc_message.trail with
-  | None -> return ()
-  | Some msg ->
+  match message.Irc_message.command with
+  | Irc_message.PRIVMSG (channel, msg) ->
     let keys = extract_pullreqs msg in
     List.map reply_of_key keys |>
     Lwt_list.iter_p (fun reply ->
-      let channels = message.Irc_message.params in
```

Comment:
PRIVMSG to multiple targets requires the targets to be comma-separated rather than space separated, so you could send such a PRIVMSG with the library but you'd have to concatenate the targets yourself.

I'm not sure how that manifests on the receiving end, i.e. if it's possible to receive a PRIVMSG with multiple targets.